controllers/provider-aws/charts/cloud-controller-manager/templates/cloud-controller-manager.yaml

```diff
@@ -0,0 +1,130 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: cloud-controller-manager
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: kubernetes
+    role: cloud-controller-manager
+spec:
+  type: ClusterIP
+  clusterIP: None
+  ports:
+  - name: metrics
+    port: {{ include "cloud-controller-manager.port" . }}
+    protocol: TCP
+  selector:
+    app: kubernetes
+    role: cloud-controller-manager
+---
+apiVersion: {{ include "deploymentversion" . }}
+kind: Deployment
+metadata:
+  name: cloud-controller-manager
+  namespace: {{ .Release.Namespace }}
+  labels:
+    garden.sapcloud.io/role: controlplane
+    app: kubernetes
+    role: cloud-controller-manager
+spec:
+  revisionHistoryLimit: 0
+  replicas: {{ .Values.replicas }}
+  selector:
+    matchLabels:
+      app: kubernetes
+      role: cloud-controller-manager
+  template:
+    metadata:
+{{- if .Values.podAnnotations }}
+      annotations:
+{{ toYaml .Values.podAnnotations | indent 8 }}
+{{- end }}
+      labels:
+        garden.sapcloud.io/role: controlplane
+        app: kubernetes
+        role: cloud-controller-manager
+    spec:
+      tolerations:
+      - effect: NoExecute
+        operator: Exists
+      containers:
+      - name: cloud-controller-manager
+        image: {{ index .Values.images "hyperkube" }}
+        imagePullPolicy: IfNotPresent
+        command:
+        - /hyperkube
+        - cloud-controller-manager
+        - --allocate-node-cidrs=true
+        - --cloud-provider={{ .Values.cloudProvider }}
```

Comment:
We can "hard-code" this to `aws`, we don't need it to be configurable anymore

Reply:
Done.